yotta/lib/target.py

```diff
@@ -484,7 +484,7 @@ def build(self, builddir, component, args, release_build=False, build_args=None,
             targets = []
         # in the future this may be specified in the target description, but
         # for now we only support cmake, so everything is simple:
-        build_type = ('Debug', 'RelWithDebInfo')[release_build]
+        build_type = ('Debug', 'RelWithDebInfo', 'Release')[(2 if strip else 1) if release_build else 0]
         if build_type:
```

Comment:
Yea looks like the else has been redundant since this: https://github.com/donatieng/yotta/commit/f4325e38d4249098eb3c1b4c4d6c7235d9dd3b83#diff-38b68db0a6b5ef9fafffc17841f6e8e6L103, a long time ago!